File: sWorld/Trunk/SWTServer/Java/sworldcommon/src/com/swt/sworld/device/domain/DeviceDoor.java
package com.swt.sworld.device.domain;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.NamedNativeQueries;
import javax.persistence.NamedNativeQuery;
import javax.persistence.Table;

@Entity
@Table(name = "swtgp_device_device_door")
@NamedNativeQueries({
    @NamedNativeQuery(
            name = "getDeviceDoor",
            query = "CALL getDeviceDoor(:groupsId)",
            resultClass = DeviceDoor.class
    )
})
public class DeviceDoor {

	@Id
	@GeneratedValue
	@Column(name = "Id", nullable = false)
	private long Id;

	@Column(name = "Name", nullable = false)
	private String Name;

	@Column(name = "Ip", nullable = false)
	private String Ip;

	@Column(name = "Port")
	private String Port;

	@Column(name = "Locked")
	private Boolean Locked;

	@Column(name = "status")
	private String status;

	@Column(name = "TimeOpen")
	private String TimeOpen;

	@Column(name = "TimeClose")
	private String TimeClose;

	@Column(name = "CreateBy")
	private String CreateBy;

	@Column(name = "CreateAt")
	private String CreateAt;

	@Column(name = "ModifiedBy")
	private String ModifiedBy;

	@Column(name = "ModifiedAt")
	private String ModifiedAt;

	@Column(name = "description")
	private String Description;

	@Column(name = "devicetimekeeping")
	private boolean deviceTimekeeping;
	
	@Column(name = "deviceofgroup")
	private boolean deviceOfGroup;
	
	public long getId() {
		return Id;
	}

	public void setId(long id) {
		Id = id;
	}

	public String getName() {
		return Name;
	}

	public void setName(String name) {
		Name = name;
	}

	public String getIp() {
		return Ip;
	}

	public void setIp(String ip) {
		Ip = ip;
	}

	public String getPort() {
		return Port;
	}

	public void setPort(String port) {
		Port = port;
	}

	public Boolean getLocked() {
		return Locked;
	}

	public void setLocked(Boolean locked) {
		Locked = locked;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public String getTimeOpen() {
		return TimeOpen;
	}

	public void setTimeOpen(String timeOpen) {
		TimeOpen = timeOpen;
	}

	public String getTimeClose() {
		return TimeClose;
	}

	public void setTimeClose(String timeClose) {
		TimeClose = timeClose;
	}

	public String getCreateBy() {
		return CreateBy;
	}

	public void setCreateBy(String createBy) {
		CreateBy = createBy;
	}

	public String getCreateAt() {
		return CreateAt;
	}

	public void setCreateAt(String createAt) {
		CreateAt = createAt;
	}

	public String getModifiedBy() {
		return ModifiedBy;
	}

	public void setModifiedBy(String modifiedBy) {
		ModifiedBy = modifiedBy;
	}

	public String getModifiedAt() {
		return ModifiedAt;
	}

	public void setModifiedAt(String modifiedAt) {
		ModifiedAt = modifiedAt;
	}

	public String getDescription() {
		return Description;
	}

	public void setDescription(String description) {
		this.Description = description;
	}

	public boolean isDeviceTimekeeping() {
		return deviceTimekeeping;
	}

	public void setDeviceTimekeeping(boolean deviceTimekeeping) {
		this.deviceTimekeeping = deviceTimekeeping;
	}

	public boolean isDeviceOfGroup() {
		return deviceOfGroup;
	}

	public void setDeviceOfGroup(boolean deviceOfGroup) {
		this.deviceOfGroup = deviceOfGroup;
	}
	
}

File: sWorld/Trunk/SWTClient/CSharp/GPCoreCommon/sWorldModel/TransportData/DeviceDoor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace sWorldModel.TransportData
{
    [DataContract]
    public class DeviceDoor
    {
        [DataMember]
        public long Id { get; set; }
        [DataMember]
        public String Name { get; set; }
        [DataMember]
        public long OrgId { get; set; }
        [DataMember]
        public long subOrgId { get; set; }
        [DataMember]
        public String Ip { get; set; }
        [DataMember]
        public String Port { get; set; }
        [DataMember]
        public bool Locked { get; set; }
        [DataMember]
        public String TimeOpen { get; set; }
        [DataMember]
        public String TimeClose { get; set; }
        [DataMember]
        public String CreateBy { get; set; }
        [DataMember]
        public String CreateAt { get; set; }
        [DataMember]
        public String ModifiedBy { get; set; }
        [DataMember]
        public String ModifiedAt { get; set; }
        [DataMember]
        public String Status { get; set; }
        [DataMember]
        public String Description { get; set; }
        [DataMember]
        public bool deviceTimekeeping { get; set; }

        [DataMember]
        public bool deviceOfGroup { get; set; }
    }
}
